ui/src/views/Dashboard.vue

```diff
@@ -1401,6 +1414,32 @@ export default {
         false
       );
     },
+    toggleSkip(app) {
+      const context = this;
+      context.loading.migrationUpdate = true;
+      nethserver.exec(
+        ["nethserver-ns8-migration/migration/update"],
+        {
+          action: "toggle-skip",
+          app: app.id
+        },
+        null,
+        function(success) {
+          context.migrationReadApps();
+          context.loading.migrationUpdate = false;
+        },
+        function(error) {
+          const errorMessage = context.$i18n.t(
+            "dashboard.error_on_skip"
+          );
+          console.error(errorMessage, error);
+          context.error.migrationUpdate = errorMessage;
+          context.loading.migrationUpdate = false;
+          context.migrationReadApps();
```

Comment:
In case of error it shouldn't be necessary to reload migration data

Reply:
But we do the same here: https://github.com/NethServer/nethserver-ns8-migration/blob/master/ui/src/views/Dashboard.vue#L1399

Can you suggest the correct patch?

Reply:
Don't remember why we are doing it. We can keep it anyway, it shouldn't be harmful

Reply:
:+1: